src/themes/mono/lv_theme_mono.c

```diff
@@ -258,13 +258,13 @@ static void theme_apply(lv_theme_t * th, lv_obj_t * obj)
 
 #if LV_USE_WIN
         /*Header*/
-        if(lv_obj_get_index(obj) == 0 && lv_obj_check_type(lv_obj_get_parent(obj), &lv_win_class)) {
+        if(lv_obj_check_type(lv_obj_get_parent(obj), &lv_win_class) && lv_obj_get_child(parent, 0) == 0) {
```

Comment:
Move `lv_obj_t * parent` out of `#if ` control and use `parent` directly below.

```suggestion
        if(lv_obj_check_type(parent, &lv_win_class) && lv_obj_get_child(parent, 0) == 0) {
```